python/lsst/sims/utils/ObservationMetaData.py

```diff
@@ -71,12 +71,6 @@ class ObservationMetaData(object):
           Analogous to m5, corresponds to the seeing in arcseconds in the bandpasses in
           bandpassName
 
-        * epoch is the epoch of the RA, Dec system used to define
```

Comment:
Because it is not being used, both in the sense that we don't have any objects defined in a different coordinate system and in the sense that, if we did, we aren't doing anything with `ObservationMetaData.epoch`, so having it there is misleading in the sense that it makes users think we are performing coordinate transformations that we are not.

See the comment on commit f1b09ce09b2c4